grabbit/src/test/groovy/com/twcable/grabbit/client/GrabbitConfigurationSpec.groovy

```diff
@@ -25,6 +25,125 @@ import spock.lang.Unroll
 class GrabbitConfigurationSpec extends Specification {
 
     @Unroll
+    def "Should return exclude paths"() {
+        when:
+        def expectedOutput = ["/content/a/b/d", "/content/a/b/e/ab", "/content/a/b/q/fg"] as Collection<String>
+        def actualOutput = GrabbitConfiguration.create(input)
+
+        then:
+        actualOutput instanceof GrabbitConfiguration
+
+        actualOutput.pathConfigurations.excludePaths.get(0) == expectedOutput
+
+        where:
+        input << [
+                """
+            {
+                "serverUsername" : "admin",
+                "serverPassword" : "admin",
+                "serverHost" : "localhost",
+                "serverPort" : "4503",
+                "deltaContent" : false,
+                "pathConfigurations" :  [
+                    {
+                        "path" : "/content/a/b",
+                        "excludePaths" : ["d", "e/ab", "q/fg"],
+                        "workflowConfigIds" : []
+                    }
+                ]
+			}
+			"""
+        ]
+    }
+
+    @Unroll
+    def "Should fail to return exclude paths"() {
+        when:
+        GrabbitConfiguration.create(input)
+
+        then:
+        final GrabbitConfiguration.ConfigurationException exception = thrown()
+        exception.errors == errors
+
+        where:
+        input          | errors
+        """
+        {
+           "serverUsername" : "admin",
+           "serverPassword" : "admin",
+           "serverHost" : "localhost",
+           "serverPort" : "4503",
+           "deltaContent" : false,
+           "pathConfigurations" :  [
+            {
+                "path" : "/content/a/b",
+                "excludePaths" : ["/a", "\\b", "./c"],
+                "workflowConfigIds" : []
+            }
+           ]
+        }
+        """ | [excludePaths: "relative paths provided cannot begin or end with /,./ or \\"]
+    }
```

Comment:
Better, that takes proper advantage of the `where:` capabilities (and shows that there's currently a bug):
```groovy
@Unroll
def "throws exception for non-relative exclude path: #excludePath"() {
    when:
    def input = """
    {
       "serverUsername" : "admin",
       "serverPassword" : "admin",
       "serverHost" : "localhost",
       "serverPort" : "4503",
       "deltaContent" : false,
       "pathConfigurations" :  [
        {
            "path" : "/content/a/b",
            "excludePaths" : ["${excludePath}"],
            "workflowConfigIds" : []
        }
       ]
    }
    """

    GrabbitConfiguration.create(input)

    then:
    GrabbitConfiguration.ConfigurationException exception = thrown()
    exception.errors == [excludePaths: "relative paths provided cannot begin or end with /,./ or \\"]

    where:
    excludePath << ["/a", "\\b", "./c"]
}
```